app/templates/index.hbs

```diff
@@ -1,123 +1,277 @@
+      </div>
+    </div>
+    <div class="layout mt-4">
+      <div class="lg:col-4">
+        <img class="img-fluid" src="/images/home/ember-routing.png" alt="" role="presentation" width="636" height="424"
+        srcset="/images/home/ember-routing-2x.png 2x">
+      </div>
+      <div class="lg:col-2">
+        <h3 class="text-lg">Routing</h3>
+        <p>Ember always cared about the URL, one of the major strengths of the web. Ember's built-in router is a best-in-class solution that incorporates async data loading with dynamic url segments and query parameters. The Ember router seamlessly supports nested urls with incremental data fetching and nested loading and error substates</p>
+      </div>
+    </div>
+    <div class="layout mt-4">
+      <div class="lg:col-4 lg:start-3">
+        <img class="img-fluid" src="/images/home/ember-data.png" alt="" role="presentation" width="636" height="424">
+      </div>
+      <div class="lg:col-2">
+        <h3 class="text-lg">Data layer</h3>
+        <p>
+          Ever new Ember app includes a fully-featured data access library called Ember Data. Ember Data allows you to access data across multiple sources at once, setup asynchronous relationships that auto-populate on access and has an identity map so that models are kept up to date across your app. Ember also supports using plain fetch() calls for data access or even GraphQL.
+        </p>
+      </div>
+    </div>
+    <div class="layout mt-4">
+      <div class="lg:col-4">
+        <img src="/images/home/ember-testing.png" alt="" role="presentation" width="636" height="424"
+        srcset="/images/home/ember-testing-2x.png 2x">
+      </div>
+      <div class="lg:col-2">
+        <h3 class="text-lg">Testing</h3>
+        <p>
+          Ember has a fully-featured, modern test harness built into every application by default. Every time you generate a component, route or any entity in an ember app we also generate an example test for that same entity. Ember supports three levels of testing: unit tests for small individual items, integration tests that are very handy for testing individual components and acceptance tests that can boot your whole app and interact with it to verify everything is working.
+        </p>
+      </div>
+    </div>
+    <div class="layout mt-4">
+      <div class="lg:col-4 lg:start-3">
+        <img class="img-fluid" src="/images/home/ember-performance.png" alt="" role="presentation" width="636" height="424"
+        srcset="/images/home/ember-performance-2x.png 2x">
+      </div>
+      <div class="lg:col-2">
+        <h3 class="text-lg">Performance</h3>
+        <p>
+          Ember is built on the Glimmer rendering engine, which compiles templates down to a highly performant DOM Virtual Machine. The Glimmer complier builds specific "programs" that are designed to create or update your node directly in the DOM without the need for a Virtual DOM, making it one of the fastest rendering technologies on the market today.
```

Comment:
```suggestion
          Ember is built on the Glimmer rendering engine, which compiles templates down to a highly performant DOM Virtual Machine. The Glimmer compiler builds specific "programs" that are designed to create or update your node directly in the DOM without the need for a Virtual DOM, making it one of the fastest rendering technologies on the market today.
```

Reply:
thanks! this is already fixed in https://github.com/ember-learn/ember-website/pull/560 👍